### PR TITLE
[IWF-1] - Add empty Xcode project.

### LINKED ISSUE #4: [IWF-1] - Create Xcode project

### PR DESCRIPTION
**Pull request by Drwalcore**
Resolves #4

### CODE REVIEW

File: Filter Reminder/Filter Reminder/SceneDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+        // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
+        // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
+        // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
+
+        // Create the SwiftUI view that provides the window contents.
+        let contentView = ContentView()
+
+        // Use a UIHostingController as window root view controller.
+        if let windowScene = scene as? UIWindowScene {
```
Comment (Drwalcore) [inline Filter Reminder/Filter Reminder/SceneDelegate.swift:26]: @jhomik look here, no storyboard removal needed.

Comment (jhomik) [reply Filter Reminder/Filter Reminder/SceneDelegate.swift:26]: awesome!

File: Filter Reminder/Filter Reminder/Filter ReminderTests/Filter_ReminderTests.swift [hunk 2]
```diff
@@ -0,0 +1,34 @@
+//
+//  Filter_ReminderTests.swift
+//  Filter ReminderTests
+//
+//  Created by Michał Czerniakowski on 04/05/2020.
+//  Copyright © 2020 drwalcore. All rights reserved.
+//
+
+import XCTest
+@testable import Filter_Reminder
```
Comment (jhomik) [inline Filter Reminder/Filter Reminder/Filter ReminderTests/Filter_ReminderTests.swift:10]: Compiler error XCode - "No such module 'Filter_Reminder'

Comment (Drwalcore) [reply Filter Reminder/Filter Reminder/Filter ReminderTests/Filter_ReminderTests.swift:10]: I will check this.

Comment (Drwalcore) [reply Filter Reminder/Filter Reminder/Filter ReminderTests/Filter_ReminderTests.swift:10]: Br0 do You even compile xD?
401thv

Zrzut ekranu 2020-05-04 18 39 37

Comment (Drwalcore) [reply Filter Reminder/Filter Reminder/Filter ReminderTests/Filter_ReminderTests.swift:10]: Try building project, then try to run unit tests with "CMD+U" shortcut.

Comment (jhomik) [reply Filter Reminder/Filter Reminder/Filter ReminderTests/Filter_ReminderTests.swift:10]: all good!

**Comment by jhomik** [comment]
closes #11

**Pull request merged**